### DISCUSSION TITLE
The browser slows down when attempting to render too many HTML nodes on the GitHub repository file view page

### DISCUSSION THREAD

**Discussion by denizguzel** [community/community#59999] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Feature Area

Issues

### Body

When I try to open the following file in the browser (Firefox Developer Edition 115.0b9, 64-bit), the browser slows down significantly while rendering the page. It appears that GitHub is attempting to render over 80,000+ HTML nodes. I didn't check all of the rendered HTML, but you can see some examples in the video.

https://github.com/oven-sh/bun/blob/main/bench/react-hello-world/react-hello-world.node.js

https://github.com/community/community/assets/10659275/d9909394-a5bc-4327-b465-4d45238378ae

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @denizguzel ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
